src/expire.c

```diff
@@ -266,8 +263,6 @@ void activeExpireCycle(int type) {
                         ttl = dictGetSignedIntegerVal(e)-now;
                         if (activeExpireCycleTryExpire(db,e,now)) {
                             expired++;
-                            /* Propagate the DEL command */
-                            postExecutionUnitOperations();
```

Comment:
why was that deleted? and missing from the other call to activeExpireCycleTryExpire?

please note that if you find bugs in the original "execution unit" PR, we may need to fix them separately, before hand, since that PR (and any followup fixups) may end up being cherry picked into 7.0.x.

Reply:
https://github.com/redis/redis/pull/11615